app/javascript/app/services/login.service.js

```diff
@@ -30,9 +30,8 @@ export async function signUp(user) {
       'Content-Type': 'application/json',
       Accept: 'application/json'
     })
-  }).then(function (response) {
-    if (response.ok) return true;
-    if (response.status === 422) return false;
+  }).then(response => {
+    if (response.ok || response.status === 422) return response.json();
```

Comment:
Just missing some comment to future developers explaining why `422` should be a valid response. 
Does it makes sense?

Reply:
Hi! It makes sense according to its definition: https://httpstatuses.com/422; there are alternatives possible (e.g. 400), this is the convention adopted by Rails.